pkg/route/routecomponenthelpers/routecomponentcontrollerhelpers.go

```diff
@@ -0,0 +1,133 @@
+package routecomponenthelpers
```

Comment:
is this a 1:1 copy?

Reply:
No, this part is new. I had to create a construct for Listers and Interfaces. Nevertheless once I move the controller, it will remove this.